public/docs/_examples/structural-directives/ts/src/app/scrap.txt

```diff
@@ -1,21 +0,0 @@
-// interesting but unused code

```

Comment:
Please restore. This is harmless and I wanted to keep it around as an idea that I had. Sure, I'll probably forget all about it (I already did). But humor me.